libs/rendering/sources/Systems/Text.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+** EPITECH PROJECT, 2023
+** r-type
+** File description:
+** Text
+*/
+
+#include "Systems/Text.hpp"
+
+rendering::system::Text::Text(
+    SparseArray<Component::Text> &texts,
+    SparseArray<Component::Position> &positions
+)
+    : _texts(texts)
+    , _positions(positions)
+{
+}
+
+rendering::system::Text::~Text()
+{
+    for (auto &font : _cache)
+        UnloadFont(font.second);
+}
+
+bool rendering::system::Text::fontIsLoaded(std::string const &path)
+{
+    return _cache.find(path) != _cache.end();
+}
+
+void rendering::system::Text::operator()()
+{
+    for (auto it = _texts.begin(); it != _texts.end(); ++it) {
+        size_t index = it.get_entity();
+
+        if (not _positions[index].has_value())
```

Comment:
je connaissais pas celui là mdr le not

Reply:
Comment rendre le code + verbeux (surtout quand t'as passé des heures à debug et que tu n'as pas vu le ptn de `!`)